docs/install/core-install.rst

```diff
@@ -201,6 +201,12 @@ your operating system's native package manager using the following commands.
 See :doc:`rocm-install-on-linux:index` for guidance on installing the ROCm
 software stack.
 
+.. note::
+
+   Omniperf will use the first version of ``Python3`` found in your system's
+   ``PATH``. If the default version of Python3 is older than 3.8, you may need to
+   update your system's ``PATH`` to point to a newer version of Python3.
+
```

Comment:
For the pip installs below, we should use:

```
python3 -m pip <...>
```

So that we ensure that the version of `pip` picked up is the same as the python3 version found in the ENV above